tests/testthat/test-assignment_linter.R

```diff
@@ -1,6 +1,11 @@
 context("assignment_linter")
 options(encoding = "UTF-8")
 test_that("returns the correct linting", {
+  # unexpected { is R language dependent
+  oldlang <- Sys.getenv("LANGUAGE")
```

Comment:
That sounds like a good idea. As an added bonus, this would automatically make all language-dependent tests safe, even if they are added in the future.